scoring/src/main/scala/ai/h2o/sparkling/ml/models/H2OWord2VecMOJOBase.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package ai.h2o.sparkling.ml.models
+
+import ai.h2o.sparkling.ml.params.H2OWord2VecExtraParams
+import ai.h2o.sparkling.sql.functions.udf
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.catalyst.expressions.GenericRowWithSchema
+import org.apache.spark.sql.expressions.UserDefinedFunction
+import org.apache.spark.sql.types.StructType
+
+import scala.collection.mutable
+
+trait H2OWord2VecMOJOBase extends H2OFeatureMOJOModel with H2OWord2VecExtraParams {
+
+  override protected def inputColumnNames: Array[String] = Array(getInputCol())
+
+  protected override def mojoUDF: UserDefinedFunction = {
```

Comment:
that udf is a shameless copy-paste from H2OMOJOPredictionWordEmbedding, it will require some bigger refactoring to reuse that trait because it requires H2OAlgorithmMOJOModel, and that's H2OFeatureMOJOModel, I even tried refactoring but it turned out too complex for that PR

Reply:
`H2OMOJOPredictionWordEmbedding` needs to be removed sooner or latter. After this PR it will become a death code. Word2Vec should be the only algorithm with word embedding predictions.